lib/dashboard.js

```diff
@@ -33,51 +33,47 @@ app.get('/dashboard', ncgUtils.authCheck, (req, res) => {
 		return res.redirect('/dashboard/');
 	}
 
-	if (!dashboardContext || configHelper.config.developer) {
+	if (!dashboardContext) {
 		dashboardContext = getDashboardContext();
 	}
 
 	res.render(path.join(__dirname, '../src/dashboard/dashboard.tmpl'), dashboardContext);
 });
 
-if (process.env.NODECG_TEST || configHelper.config.developer) {
-	app.get('/nodecg-api.min.js', (req, res) => {
-		res.sendFile(path.join(process.env.NODECG_TEST ? INSTRUMENTED_PATH : BUILD_PATH, 'nodecg-api.min.js'));
-	});
-
-	app.get('/nodecg-api.min.js.map', (req, res) => {
-		res.sendFile(path.join(process.env.NODECG_TEST ? INSTRUMENTED_PATH : BUILD_PATH, 'nodecg-api.min.js.map'));
-	});
-
-	if (process.env.NODECG_TEST) {
-		log.warn('Serving instrumented (and also unbundled) files for testing');
-		app.get('/*', (req, res, next) => {
-			const resName = req.params[0];
-			if (!resName.startsWith('dashboard/') && !resName.startsWith('instance/')) {
-				return next();
-			}
+app.get('/nodecg-api.min.js', (req, res) => {
+	res.sendFile(path.join(process.env.NODECG_TEST ? INSTRUMENTED_PATH : BUILD_PATH, 'nodecg-api.min.js'));
+});
 
-			const fp = path.join(INSTRUMENTED_PATH, resName);
-			if (fs.existsSync(fp)) {
-				return res.sendFile(fp, err => {
-					/* istanbul ignore next */
-					if (err && !res.headersSent) {
-						return next();
-					}
-				});
-			}
+app.get('/nodecg-api.min.js.map', (req, res) => {
+	res.sendFile(path.join(process.env.NODECG_TEST ? INSTRUMENTED_PATH : BUILD_PATH, 'nodecg-api.min.js.map'));
+});
 
+if (process.env.NODECG_TEST) {
+	log.warn('Serving instrumented (and also unbundled) files for testing');
+	app.get('/*', (req, res, next) => {
+		const resName = req.params[0];
+		if (!resName.startsWith('dashboard/') && !resName.startsWith('instance/')) {
 			return next();
-		});
-	} else {
-		log.warn('Serving unbundled files for development');
-	}
+		}
 
-	app.use('/', express.static(SRC_PATH));
+		const fp = path.join(INSTRUMENTED_PATH, resName);
+		if (fs.existsSync(fp)) {
+			return res.sendFile(fp, err => {
+				/* istanbul ignore next */
+				if (err && !res.headersSent) {
+					return next();
+				}
+			});
+		}
+
+		return next();
+	});
 } else {
-	app.use('/', express.static(BUILD_PATH));
+	log.warn('Serving unbundled files for development');
```

Comment:
I think this will be logged every time NodeCG is launched without NODECG_TEST environment variable.

Reply:
Good catch, I had meant to remove this log line completely. Fixed.